### ISSUE TITLE
Keep feature "accept legal terms"?

### ISSUE THREAD

**Issue by moritzschepp** [coneda/kor#38]
New users have to accept a set of usage terms defined by the admin. Since we have the possibility of a guest account, that feature doesn't make sense (in that situation). Should we keep it? If yes: how should we handle the guest user?

**Comment by wuebbena** [comment]
I would really like to keep this feature. Why? You have to differentiate between the content which is visible with the guest account and the other which is often subject to legal restrictions. For this case we do need a possibility to show/accept the usage terms. Or is this ticket just about to disable the usage terms for the guest accounts? If yes: get rid of it!

**Comment by moritzschepp** [comment]
Right, not every content is always free. So this means that we can actually leave the feature like it is. As we are separating API and frontend more and more, could we move that functionality to the frontend? So an API user wouldn't have to accept any terms, while a web user would?

**Comment by wuebbena** [comment]
To separate it in that way would be a good idea. The usage terms for the API case will be negotiate at another place so it's not necessary to have this feature in the backend.

**Comment by wuebbena** [comment]
usage terms -> web user.
